tests/integration/api/test_api.py

```diff
@@ -127,9 +127,34 @@ def test_post_hash_comparison_failure(
     test_hash_comparison_without_file: Path,
 ) -> None:
     """Returns fixture to test file."""
+    with TestClient(populated_app) as app:
+        with open(test_hash_comparison_without_file, "rb") as testFile:
+            response = app.post(
+                "/metadata/hashComparison",
+                files={
+                    "fileInput": testFile,
+                },
+                params={
+                    "sourceIsoName": "Windows",
+                },
+            )
+            assert response.status_code == 404
+
+
```

Comment:
I don't see any 200 status codes. Are you guys asserting positive responses as well?

Reply:
Tory said that there was no point testing for the 200 status code since it doesn't give us any additional information.